### ISSUE TITLE
visualization

### ISSUE THREAD

**Issue by caiyongqi** [ken2576/vision-nerf#6]
Could you release the code that renders the visualization video and calculates metrics? It will save me a lot of time.
Thanks!

**Comment by ken2576** [comment]
You can use the script provided by PixelNeRF
https://github.com/sxyu/pixel-nerf/blob/master/eval/calc_metrics.py
And for rendering, `eval.py` and `gen_real.py` could be helpful.

**Comment by SeaBird-Go** [comment]
@ken2576 Hello, when I trained the model by the default settings on the SRN chairs dataset, I found my results are very bad when compared with your pretrained model. Could you help me why this happens?

Results generated by your pretrained model:
![图片](https://user-images.githubusercontent.com/117327481/230606435-98772ae9-0a8b-4388-a4af-126b96d8d445.png)

Results generated by my trained model:
![图片](https://user-images.githubusercontent.com/117327481/230606485-40fb9266-55b0-4f06-9524-b3e2cc2f8444.png)

**Comment by ken2576** [comment]
Hi

What is the batch size you used for the training?
We found that larger batch size helps with this issue.
The setting for the pretrained model is 16 A100s and each with a batch size of 8.
If you can't fit all of them in your memory, you could consider accumulating the gradients to achieve a larger batch.
But for that you might need to implement it yourself, since I didn't include code for that.
